extensions/guacamole-auth-cas/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET" : ""
```

Comment:
I think that helps, although the redirect happens quickly enough that it's a little hard to tell.  I'll at least remove the empty translation information from my extension.